lib/openc_bot/helpers/register_methods.rb

```diff
@@ -17,14 +17,14 @@ def allowed_hours
         elsif self.const_defined?('TIMEZONE')
           # See https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for definitions/examples
           # eg TIMEZONE = "America/Panama"
-          tz = TZInfo::Timezone.get(self.const_get('TIMEZONE'))
-          non_working_hours_starts = tz.local_to_utc(Time.parse("#{Date.today} 18:00")).hour
-          non_working_hours_ends = tz.local_to_utc(Time.parse("#{Date.today} 08:00")).hour
-          if non_working_hours_starts < non_working_hours_ends
-            (non_working_hours_starts..non_working_hours_ends).to_a
-          else
-            (non_working_hours_starts..24).to_a + (0..non_working_hours_ends).to_a
-          end
+          # tz = TZInfo::Timezone.get(self.const_get('TIMEZONE'))
+          # non_working_hours_starts = tz.local_to_utc(Time.parse("#{Date.today} 18:00")).hour
+          # non_working_hours_ends = tz.local_to_utc(Time.parse("#{Date.today} 08:00")).hour
+          # if non_working_hours_starts < non_working_hours_ends
+          #   (non_working_hours_starts..non_working_hours_ends).to_a
+          # else
+          (18..24).to_a + (0..8).to_a
+          # end
```

Comment:
Can we just delete the code instead of commenting it out? We'll still have it in the git history!